python/lsst/ts/cbp/mock_server.py

```diff
@@ -120,16 +140,11 @@ def __init__(self, log=None):
         super().__init__(
             name="CBP Mock Server",
             host=tcpip.LOCAL_HOST,
-            port=9999,
-            connect_callback=self.connect_callback,
+            port=0,
             log=self.log,
+            terminator=terminator,
```

Comment:
This is not necessary. The default value for `terminator` is `tcpip.DEFAULT_TERMINATOR` which has exactly the same value of `b"\r\n"`.